tests/version_1.0/behavior/sibling_files.wdl

```diff
@@ -0,0 +1,59 @@
+version 1.0
+
+workflow wf {
+    input {
+        File first
+        File second
+        String first_as_string = first
+        String second_as_string = second
+    }
+    call t {
+        input:
+            first=first_as_string,
+            second=second_as_string
+    }
+    output {
+        Boolean equal = t.equal
+        # String out1 = t.out1
+        # String out2 = t.out2
+    }
+}
+
+task t {
+    input {
+        String first
+        String second
+    }
+
+    String first_basename = basename(first)
+    String second_basename = basename(second)
+    command <<<
+        function f {
+            FILE=$1
+            if [[ $FILE == *"toilfile"* ]]; then
```

Comment:
I don't think the `toilfile:` URIs are meant to be visible inside the tasks.

Also, the WDL spec doesn't say anything about `toilfile`, so it seems weird that it would appear in the conformance tests. I feel like you just want to look at `dirname ~{first}` and `dirname ~{second}` and make sure they are equal?